src/env.cc

```diff
@@ -259,6 +259,19 @@ std::vector<size_t> IsolateData::Serialize(SnapshotCreator* creator) {
 #undef VY
 #undef VS
 #undef VP
+
+#define V(PropertyName, TypeName)                                              \
+  do {                                                                         \
+    Local<TypeName> field = PropertyName();                                    \
+    if (!field.IsEmpty()) {                                                    \
+      indexes.push_back(creator->AddData(field));                              \
+    } else {                                                                   \
+      indexes.push_back(0);                                                    \
+    }                                                                          \
```

Comment:
```suggestion
    if (field.IsEmpty()) {                                                     \
      indexes.push_back(0);                                                    \
    } else {                                                                   \
      indexes.push_back(creator->AddData(field));                              \
    }                                                                          \
```
No negation = easier to read.